Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -1380,7 +1380,6 @@
   - SecurityBelt
   - SecurityWebbing
   - SecurityWebbingAlt  # Moffstation - Alt sec webbing
-  - ShoulderHolster # Moffstation - (Security Loadout Improvements) - Filled shoulder holster as a sidegrade to the belt, rather than detectives having to start with the shoulder holster.
```

Comment:
No more secoff / HoS shoulder holster loadout?

Reply:
I dunno if anyone does, I just am pointing it out as a conceptually separate change you didn't mention in the descripto.